cmd/dep/godep_importer.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/golang/dep"
+	fb "github.com/golang/dep/internal/feedback"
+	"github.com/golang/dep/internal/gps"
+	"github.com/pkg/errors"
+)
+
+const godepJSONName = "Godeps.json"
+
+type godepImporter struct {
+	json godepJSON
+
+	logger  *log.Logger
+	verbose bool
+	sm      gps.SourceManager
+}
+
+func newGodepImporter(logger *log.Logger, verbose bool, sm gps.SourceManager) *godepImporter {
+	return &godepImporter{
+		logger:  logger,
+		verbose: verbose,
+		sm:      sm,
+	}
+}
+
+type godepJSON struct {
+	Name    string         `json:"ImportPath"`
```

Comment:
yeah, doesn't seem to be useful until we decide to make it mandatory for a valid config file.
Removing :)